website/views/shared/tasks/lists.jade

```diff
@@ -24,9 +24,14 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
 
           // Add New
           form.task-add(name='new{{list.type}}form', ng-hide='obj._locked', ng-submit='addTask(obj[list.type+"s"],list)')
-            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', required)
+            textarea(rows='6', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', required)
+            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', ng-if='!list.bulk', required)
             button(type='submit', ng-disabled='new{{list.type}}form.$invalid')
               span.glyphicon.glyphicon-plus
+            span.help-block
```

Comment:
I'll take your CSS as-is. It's pretty much exactly what I had planned to do, but I wanted to get to bed and decided to post the first draft for feedback to make sure I was on a track people agreed with.